libraries/AP_BattMonitor/AP_BattMonitor_AD7091R5.cpp

```diff
@@ -0,0 +1,236 @@
+#include "AP_BattMonitor_AD7091R5.h"
+
+/**
+ * @brief You can use it to Read Current and voltage of 1-3 batteries.
```

Comment:
please expand this explanation to show how to configure, or give an example setup with parameter names and values and how it would be connected.

I also would like you to add a #define for a AD7091R5_BASE_PIN, which is the starting pin number. I suggest a value of 50, so the pins will be 50, 51, 52, 53 etc. The reason for that is we would like to later add the AD7091R5 as a generic ADC input system, and we'll need to be able to distinguish pin numbers from the built-in analog pins. Starting at 50 will mean the two ranges don't overlap.